docs/management/connectors/action-types/opsgenie.asciidoc

```diff
@@ -0,0 +1,173 @@
+User::    The display name of the owner.
+Note::    Additional information for the alert.
+
+[float]
+[[opsgenie-action-create-alert-json-configuration]]
+====== JSON editor
+
+A JSON editor is provided as an alternative to the form view. The editor also supports additional fields not shown in the form. The editor view supports all of the forms configuration properties but as lowercase keys as https://docs.opsgenie.com/docs/alert-api#create-alert[described in the Opsgenie API documentation]. The editor view supports the following additional properties.
+
+responders::  The entities to receive notifications about the alert.
+visibleTo::   The teams and users that the alert will be visible to without sending a notification to them.
+actions::   The custom actions available to the alert.
+details::   The custom properties of the alert.
+
+[float]
+[[opsgenie-action-create-alert-json-example-configuration]]
+Example JSON editor contents
+
+[source,json]
+--
+{
+  "message": "An example alert message",
+  "alias": "Life is too short for no alias",
+  "description":"Every alert needs a description",
+  "responders":[
+      {"id":"4513b7ea-3b91-438f-b7e4-e3e54af9147c", "type":"team"},
+      {"name":"NOC", "type":"team"},
+      {"id":"bb4d9938-c3c2-455d-aaab-727aa701c0d8", "type":"user"},
+      {"username":"trinity@opsgenie.com", "type":"user"},
+      {"id":"aee8a0de-c80f-4515-a232-501c0bc9d715", "type":"escalation"},
+      {"name":"Nightwatch Escalation", "type":"escalation"},
+      {"id":"80564037-1984-4f38-b98e-8a1f662df552", "type":"schedule"},
+      {"name":"First Responders Schedule", "type":"schedule"}
+  ],
+  "visibleTo":[
+      {"id":"4513b7ea-3b91-438f-b7e4-e3e54af9147c","type":"team"},
+      {"name":"rocket_team","type":"team"},
+      {"id":"bb4d9938-c3c2-455d-aaab-727aa701c0d8","type":"user"},
+      {"username":"trinity@opsgenie.com","type":"user"}
+  ],
+  "actions": ["Restart", "AnExampleAction"],
+  "tags": ["OverwriteQuietHours","Critical"],
+  "details":{"key1":"value1","key2":"value2"},
+  "entity":"An example entity",
+  "priority":"P1"
+}
+--
+
+[float]
+[[opsgenie-action-close-alert-configuration]]
+===== Close alert configuration
+
+The close alert action has the following configuration properties.
+
+Alias::   The identifier for the alert. The `Alias` is used for https://support.atlassian.com/opsgenie/docs/what-is-alert-de-duplication/[alert de-duplication]. The alias must match the value used when creating the alert.
+Note::    Additional information for the alert.
+Source::  The display name of the source.
+User::    The display name of the owner.
+
+[float]
+[[configuring-opsgenie]]
+==== Configure an Opsgenie account
+
+Opsgenie offers https://www.atlassian.com/software/opsgenie/try[free trial instances], which you can use to test creating and closing alerts.
+
+After obtaining an Opsgenie instance, https://support.atlassian.com/opsgenie/docs/create-a-default-api-integration/[configure the API integration].
```

Comment:
```suggestion
After creating an Opsgenie instance, https://support.atlassian.com/opsgenie/docs/create-a-default-api-integration/[configure the API integration].
```

Reply:
Per my other suggestions, I think it's a good idea to make it clear we're sending them outside our docs here too. For example:
```suggestion
After obtaining an Opsgenie instance, configure the API integration. For details, refer to the https://support.atlassian.com/opsgenie/docs/create-a-default-api-integration/[Opsgenie documentation].
```